spec/models/assignment_spec.rb

```diff
@@ -92,43 +92,158 @@
   end
 
   describe ".for_presentation" do
-    let(:split) { FactoryBot.create(:split) }
+    it "filters for one reason of underlying scopes (decision overrides)" do
```

Comment:
@smudge this is attempting to establish some confidence that the composed scopes still reflect the behavior of the underlying scopes `and`ed-together without boiling the combinatorial ocean, because if they were `or`ed-together, this filtering wouldn't work.